find-mmc-bypartlabel

```diff
@@ -39,12 +39,21 @@ while [ ! -d /sys/class/block/mmcblk0p* ] && [ ! -d /sys/class/block/sda ] ; do
         exit 1
     fi
 done
-
-for mmc_sysfs in /sys/class/block/mmcblk0p*/ /sys/class/block/sda*/; do
-	if grep -q -w PARTNAME="$1" "$mmc_sysfs"/uevent 2> /dev/null; then
-		FIMAGE_DEV_NAME=$(echo $mmc_sysfs | cut -d "/" -f 5)
-		echo "/dev/$FIMAGE_DEV_NAME"
-		exit 0
+FIMAGE_DEV_NAME=""
+i=0
+while [ -z "$FIMAGE_DEV_NAME" ]; do
+	for mmc_sysfs in /sys/class/block/mmcblk0p*/ /sys/class/block/sda*/; do
+		if grep -q -w PARTNAME="$1" "$mmc_sysfs"/uevent 2> /dev/null; then
+			FIMAGE_DEV_NAME=$(echo $mmc_sysfs | cut -d "/" -f 5)
+			echo "/dev/$FIMAGE_DEV_NAME"
+			exit 0
+		fi
+	done
+	sleep 0.1
+	(( i++))
```

Comment:
This is bash syntax in POSIX this should be i=$((i+1))

Reply:
Was not sure which syntax to use as the script already contains bashisms.

Reply:
Thanks for review!

Reply:
I don't see any bashism in the script.